modules/installation-configuration-parameters.adoc

```diff
@@ -798,58 +828,108 @@ When running on ARM based AWS instances, ensure that you enter a region where AW
+|Valid link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html[AWS EBS volume type],
+such as `io1`.
+
+|controlPlane:
+  platform:
+    aws:
+      rootVolume:
+        kmsKeyARN:
 |The Amazon Resource Name (key ARN) of a KMS key. This is required to encrypt operating system volumes of control plane nodes with a specific KMS key.
 |Valid link:https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html[key ID and the key ARN].
 
-|`controlPlane.platform.aws.type`
+|controlPlane:
+  platform:
+    aws:
+      type:
 |The EC2 instance type for the control plane machines.
 |Valid AWS instance type, such as `m6i.xlarge`. See the *Supported AWS machine types* table that follows.
 //add an xref when possible
 
-|`controlPlane.platform.aws.zones`
+|controlPlane:
+  platform:
+    aws:
+      zones:
 |The availability zones where the installation program creates machines for the
 control plane machine pool.
 |A list of valid AWS availability zones, such as `us-east-1c`, in a link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
-|`controlPlane.aws.region`
+|controlPlane:
+  aws:
+    region:
 |The AWS region that the installation program creates control plane resources in.
 |Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
 
-|`platform.aws.amiID`
+|platform:
+  aws:
+    amiID:
 |The AWS AMI used to boot all machines for the cluster. If set, the AMI must
 belong to the same region as the cluster. This is required for regions that require a custom {op-system} AMI.
 |Any published or custom {op-system} AMI that belongs to the set AWS region. See _{op-system} AMIs for AWS infrastructure_ for available AMI IDs.
 
-|`platform.aws.hostedZone`
+|platform:
+  aws:
+    hostedZone:
 |An existing Route 53 private hosted zone for the cluster. You can only use a pre-existing hosted zone when also supplying your own VPC. The hosted zone must already be associated with the user-provided VPC before installation. Also, the domain of the hosted zone must be the cluster domain or a parent of the cluster domain. If undefined, the installation program creates a new hosted zone.
 |String, for example `Z3URY6TWQ91KVV`.
 
-|`platform.aws.hostedZoneRole`
+|platform:
+  aws:
+    hostedZoneRole:
 |An Amazon Resource Name (ARN) for an existing IAM role in the account containing the specified hosted zone. The installation program and cluster operators will assume this role when performing operations on the hosted zone. This parameter should only be used if you are installing a cluster into a shared VPC.
 |String, for example `arn:aws:iam::1234567890:role/shared-vpc-role`.
 
-|`platform.aws.serviceEndpoints.name`
-|The AWS service endpoint name. Custom endpoints are only required for cases
+|platform:
+  aws:
+    serviceEndpoints:
+      - name:
+        url:
+|The AWS service endpoint name and URL. Custom endpoints are only required for cases
```

Comment:
If this is all you changed since the first peer review, then these changes look good to me!